ingest/muxer.go

```diff
@@ -270,6 +270,7 @@ func newIngestMuxer(c MuxerConfig) (*IngestMuxer, error) {
 
 	// connect up the chancacher
 	gob.Register(&entry.Entry{})
```

Comment:
should these be at the package level in an `init()` so that we don't run into something similar later when we try to use the package in an new and interesting way?